docs/farming-&-staking/farming/common-problems.mdx

```diff
@@ -0,0 +1,129 @@
+
+
+🟠 This warning message should not affect your rewards or the operation of your node. It's just a log message indicating the normal operation of the network.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+</TabItem>
+
+
+<TabItem value="farmer" label="Farmer">
+
+<details>
+<summary><small>ERROR subspace_farmer::farmer_cache: Failed to store piece in cache, there was no space piece_index=&lt;piece&gt;</small></summary>
+
+
+🔴 This indicates that there is not enough space in the cache to store a new piece of data.
+
+The drive where you are storing your plot is out of space. Reduce the plot size and restart the farmer. Additionally, ensure that you are saving the plot to the correct drive and that it is properly mounted.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+</TabItem>
+
+
+<TabItem value="farming-cluster" label="Farming Cluster">
+
+<details>
+<summary><small>WARN \{farm_index=3}:\{sector_index=1036}: subspace_farmer::single_disk_farm::plotting: Failed to plot sector, retrying in 1s error=Low-level plotting error: Received only &lt;sector&gt; sector bytes out of &lt;bytes&gt; expected bytes</small></summary>
+
+
+🟠 This is a known issue that is actively being investigated. To help mitigate these errors, run your NATS component on the same system as your Controller, Cache, and Plotter components.
+
+<small>Last Confirmed: gemini-3h-2024-oct-03</small>
+</details>
+
+</TabItem>
+
+</Tabs>
+
+## Unexpected Behavior
+
+These unexpected behaviors are not linked to specific components. In this section, we will outline common issues and provide any available remediation steps to address them.
+
+:::caution[Handling No Output and Missing Error Codes]
+
+If you encounter a situation where the node produces no output and does not display any error code, it is likely that you simply need to install the latest version of the Visual C++ Redistributable package, which can be found [here](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).
+
+:::
+
+:::tip[Farmer is stuck on plotting and has not made any progress for several hours.]
+
+Try restarting your node or farmer. We've observed that when creating larger plots, the process may sometimes seem stalled, but it typically resumes automatically after a while.
+
+:::
+
+:::tip[Rewards have not been earned after several days of farming.]
+
+- Ensure you are using the latest release of either [Space Acres](https://github.com/autonomys/space-acres/releases) or the [Advanced CLI](https://github.com/autonomys/subspace/releases).
+- Confirm that your farmer is active and on the highest block by checking our [telemetry](https://telemetry.subspace.network) server.
+- Check your balance using [Astral Bock Explorer](https://astral.autonomys.xyz) or the [Polkadot Explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-0.gemini-3h.subspace.network%2Fws#/explorer)
+
+:::
+
+
+:::tip[Still experiencing issues?]
+
+<details>
+
+<summary>
```

Comment:
I know this is borrowed from the original page implementation, but I think we can drop it. Let's just provide the links to the forum, farmer-support /farmer-chat, please